setup.py

```diff
@@ -134,6 +134,9 @@ def make_pybind11_extension(
     rpathflag = '-Wl,-rpath,%s/lib' % sys.exec_prefix
     if "CONDA_PREFIX" in os.environ:
         include_dirs.append(os.path.join(os.environ["CONDA_PREFIX"], "include"))
+    else:
+        include_dirs.append(os.path.join("/usr/include/scotch"))
+        include_dirs.append(os.path.join("/usr/include/metis"))
```

Comment:
SCOTCH should provide metis.h.  We should use SCOTCH's metis.h instead of METIS's.

If you failed to build without the METIS package your there's something wrong with you environment.  The conda environment doesn't have METIS and it builds well.  You probably miss The METIS compatible layer of SCOTCH.